File: python/opencv/opencv-old/image/rgb.py
# -*- coding: utf-8 -*-
import cv2

def main():
    im = cv2.imread("test.jpg",0)               # 画像をグレースケールで読み込み
    im = cv2.cvtColor(im, cv2.COLOR_GRAY2RGB)   # グレースケール画像をRGB色空間に変換
    cv2.imshow("Show",im)                       # ウィンドウの表示
    cv2.waitKey(0)
    cv2.destroyAllWindows()

if __name__ == '__main__':
    main()

File: cpp/opencv/image/rgb.cpp
#include <iostream>
#include "opencv2\opencv.hpp"

using namespace cv;
  
int main(void)
{
	Mat rgb;
	Mat hsv = imread("test.png");	  // 画像の取得
	cvtColor(hsv, rgb, CV_HSV2RGB); //HSVからRGBに変換
	imshow("Show image", rgb);	    // 画像の表示
	waitKey(0);			                // 入力待機
	return 0;
}
